tutorials/Sessions.md

```diff
@@ -0,0 +1,80 @@
+# Sessions
+Stytch user sessions are identified by a session token and a JWT (JSON Web Token) that are authenticated on, and returned from, our authentication endpoints. The Stytch iOS SDK automatically persists these tokens and appends them to network requests as required, to make interacting with Stytch's authentication flows as simple as possible, and provides both Automatic and Manual session management helpers.
+
+## SDK Sessions At-A-Glance
+### Session Data Persistence
+The Stytch iOS SDK persists the session token and JWT to device by saving them to the Keychain, after encrypting them using AES256-GCM. The actual session and user data is stored in-memory only.
```

Comment:
Is the AES256-GCM part correct for iOS? I don't know what it uses when saving to Keychain